Model/Export/AttributeOptionExportManager.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace StoreKeeper\StoreKeeper\Model\Export;
+
+use Magento\Catalog\Api\Data\CategoryInterface;
+use StoreKeeper\StoreKeeper\Model\Export\AbstractExportManager;
+use Magento\Framework\Locale\Resolver;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Store\Api\StoreConfigManagerInterface;
+use Magento\Eav\Api\AttributeRepositoryInterface;
+use StoreKeeper\StoreKeeper\Model\Export\CsvFileContent;
+use Magento\Eav\Model\Attribute;
+use Magento\Eav\Api\Data\AttributeOptionInterface;
+use StoreKeeper\StoreKeeper\Helper\Api\Auth;
+use Magento\Swatches\Model\Swatch;
+use Magento\Swatches\Helper\Media as SwatchHelper;
+use Magento\Swatches\Model\ResourceModel\Swatch\CollectionFactory as SwatchCollectionFactory;
+
+class AttributeOptionExportManager extends AbstractExportManager
+{
+    const HEADERS_PATHS = [
+        'path://name',
+        'path://label',
+        'path://translatable.lang',
+        'path://is_main_lang',
+        'path://is_default',
+        'path://image_url',
+        'path://attribute.name',
+        'path://attribute.label'
+    ];
+    const HEADERS_LABELS = [
+        'Name',
+        'Label',
+        'Language',
+        'Is main language',
+        'Is default',
+        'Image URL',
+        'Attribute name',
+        'Attribute label'
+    ];
+
+    private Attribute $attribute;
+    private Auth $authHelper;
+    private SwatchHelper $swatchHelper;
+    private SwatchCollectionFactory $swatchCollectionFactory;
+
+    public function __construct(
+        Resolver $localeResolver,
+        StoreManagerInterface $storeManager,
+        StoreConfigManagerInterface $storeConfigManager,
+        Attribute $attribute,
+        Auth $authHelper,
+        SwatchHelper $swatchHelper,
+        SwatchCollectionFactory $swatchCollectionFactory
+    ) {
+        parent::__construct($localeResolver, $storeManager, $storeConfigManager, $authHelper);
+        $this->attribute = $attribute;
+        $this->swatchHelper = $swatchHelper;
+        $this->swatchCollectionFactory = $swatchCollectionFactory;
+    }
+
+    /**
+     * @param array $attributeOptions
+     * @return array
+     */
+    public function getAttributeOptionExportData(array $attributeOptions): array
+    {
+        $result = [];
+        foreach ($attributeOptions as $attributeOption) {
+            $attribute = $this->getAttribute($attributeOption);
+            $attributeData = $this->getAttributeData($attribute);
+            $attributeOptionId = $attributeOption->getId();
+            $attributeOptionData = $this->getAttributeOptionData($attribute, $attributeOption);
+            $data = [
+                isset($attributeData['attribute_code']) ? $attributeData['attribute_code'] . '_' . $attributeOptionId : null, //path://name'
```

Comment:
can `$attributeData['attribute_code']` it be null ?

Reply:
@ikulis No, In method deffinition it awaits attribute entity as input param, which guarantees that we will have its code in 100$ of cases. https://github.com/storekeeper-company/magento2-plugin/blob/d5f743ff78bc6ede57b11b68ff3bfb59dbaed474/Model/Export/AttributeOptionExportManager.php#L103